scripts/install.py

```diff
@@ -663,6 +589,13 @@ def have_sw_package(pkg):
 
 def install_gc3pie_from_github(venv_dir, features,
                                repo=default.GC3PIE_REPO_URL):
+    if default.ANACONDA:
+        die(os.EX_SOFTWARE, "Anaconda install from source not supported",
```

Comment:
Why is this not supported? As far as I can see, packages can be installed into conda envs via `pip`::

    $ export PATH=$HOME/w/anaconda/4.1.1/bin:$PATH
    $ python -V
    Python 2.7.12 :: Anaconda 4.1.1 (64-bit)
    $ conda env list
    # conda environments:
    #
    root                  *  /home/rmurri/w/anaconda/4.1.1

    $ conda create --name gc3pie --clone root
    Source:      /home/rmurri/w/anaconda/4.1.1
    Destination: /home/rmurri/w/anaconda/4.1.1/envs/gc3pie
    ...
    #
    # To activate this environment, use:
    # $ source activate gc3pie
    #
    # To deactivate this environment, use:
    # $ source deactivate
    #
    $ source activate gc3pie
    (gc3pie) $ pip install -e '.'
    Obtaining file:///home/rmurri/w/gc3pie
        Complete output from command python setup.py egg_info:

        The required version of setuptools (>=21.0.0) is not available,
        and can't be installed while this script is running. Please
        install a more recent version first, using
        'easy_install -U setuptools'.

        (Currently using setuptools 20.9.0 (/home/rmurri/.local/lib/python2.7/site-packages))

        ----------------------------------------
    Command "python setup.py egg_info" failed with error code 2 in /home/rmurri/w/gc3pie/
    (gc3pie) $ pip install --upgrade setuptools
    ...
    Successfully installed setuptools-32.3.1
    (gc3pie) $ pip install -e '.'
    Obtaining file:///home/rmurri/w/gc3pie
    ...
    Successfully installed coloredlogs-5.2 gc3pie humanfriendly-2.2.1 lockfile-0.12.2 paramiko-2.1.1 parsedatetime-2.1 prettytable-0.7.2 pyCLI-2.0.3 python-daemon-2.1.2
    (gc3pie) $ gservers   --version
    gservers development version (SVN $Revision$)